internalsdk/android.go

```diff
@@ -616,6 +622,10 @@ func run(configDir, locale string, settings Settings, session PanickingSession)
 		log.Debugf("Chat enabled? %v", chatEnabled)
 		session.SetChatEnabled(chatEnabled)
 
+		authEnabled := runner.FeatureEnabled("auth", common.ApplicationVersion)
```

Comment:
@atavism Did we add this flag on flashlight?

Reply:
Did we add this to [global.ymal.tmpl](https://github.com/getlantern/flashlight/blob/a736060c32caf80ad772507c1fb762547489d8fe/embeddedconfig/global.yaml.tmpl#L1)

Reply:
@jigar-f We had an "auth" feature flag from before specifically to add username/password support that's no longer being used. I'm just re-using that